app/views/rosters/index.haml

```diff
@@ -1,6 +1,6 @@
 %h1.title Rosters
 
-%table.table.auto-width.col-xs-offset-3
+%table.table.auto-width.col-xs-offset-1
```

Comment:
This looks worse, but it was a little off anyways.

I don't think we should use an offset that is a fixed number of columns to make up for the table not being  the full width.

The `.table` class actuall does make the "`width`" `100%`, but all the columns have so little information in them that they don't fill up the table.